eng/scripts/Update-PkgVersion.ps1

```diff
@@ -0,0 +1,33 @@
+
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $PackageName,
+  [string] $NewVersionString,
+  [string] $ReleaseDate
+)
+
+. (Join-Path $PSScriptRoot ".." common scripts common.ps1)
+
+$pkgProperties = Get-PkgProperties -PackageName $PackageName
+$packageVersion = $pkgProperties.Version
+
+$packageSemVer = [AzureEngSemanticVersion]::new($packageVersion)
+
+if ([System.String]::IsNullOrEmpty($NewVersionString))
+{
+    $packageSemVer.IncrementAndSetToPrerelease()
+    & "${EngCommonScriptsDir}/Update-ChangeLog.ps1" -Version $packageSemVer.ToString() `
+    -ChangelogPath $pkgProperties.ChangeLogPath -Unreleased $True
+}
+else
+{
+    $packageSemVer = [AzureEngSemanticVersion]::new($NewVersionString)
+    & "${EngCommonScriptsDir}/Update-ChangeLog.ps1" -Version $packageSemVer.ToString() `
+    -ChangelogPath $pkgProperties.ChangeLogPath -Unreleased $False `
+    -ReplaceLatestEntryTitle $True -ReleaseDate $ReleaseDate
+}
+
+$pkgVersion, $versionFile = Get-Version -pkgPath $pkgProperties.DirectoryPath
+$versionFileContent = Get-Content -Path $versionFile
```

Comment:
Lets do "-Raw" get and "-NoNewLine" for set so that we don't impact the line endings in the file.